test/int.jl

```diff
@@ -190,6 +190,16 @@ end
         end
     end
 end
+@testset "bit rotations" begin
```

Comment:
How about throwing some tests for some specific literal rotations in here? It would make sense to me to pick some bit pattern and test that rotating it by various numbers of bits works. Testing `ror` and `rol` against each other is fine, but if they both just return zero all the time, these tests would still pass. Generic tests are great, but sometimes it's good to also test a few very concrete things that you can check with your eyeballs and common sense.

Reply:
Changed to specific tests, but I wasn't sure how to generalize to all `BitInteger_types` cleanly.